.github/workflows/python-app.yml

```diff
@@ -13,6 +13,22 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Set up Poetry
+        uses: Gr1N/setup-poetry@v4
+      - name: Install dependencies
+        run: poetry install
+      - name: Run app
+        run: poetry run python src/main.py ${{ secrets.GITHUB_TOKEN }}
+
+  prettier:
```

Comment:
Please split this out into a separate workflow called `docs.yml`, and call the workflow `lint`. Move your actual `lint` job above `build` and have `build` depend on it so that we do not build unless the lint is successful.